### DISCUSSION TITLE
Can this be used with fluent translations?

### DISCUSSION THREAD

**Discussion by razzeee** [i18next/next-i18next#1697] [Q&A] [answered]
I'm wondering, if this is able to handle fluent translation files? https://github.com/i18next/react-i18next/blob/master/example/v9.x.x/react-fluent/src/i18n.js#L8

https://projectfluent.org/

**Comment by isaachinman** [comment, marked as answer]
Yes, all i18next plugins/features can be used with `next-i18next`.

**Reply by razzeee** [reply]
Would that mean, that I need to import something like https://github.com/i18next/react-i18next/blob/master/example/v9.x.x/react-fluent/src/i18n.js in my `_app.ts`?

**Reply by isaachinman** [reply]
Yes, you would need to pass the plugin into the config [via the `use` array](https://github.com/isaachinman/next-i18next#options).

**Reply by razzeee** [reply]
Doesn't seem like I'm able to figure it out

![image](https://user-images.githubusercontent.com/5943908/155371706-ebb519a7-5ce7-4732-8ec7-36901874c860.png)

**Reply by isaachinman** [reply]
You need to pass the plugin via `next-i18next.config.js`, not the global `i18n` object.
